bigquery/snippets/managedwriter/integration_test.go

```diff
@@ -84,8 +84,16 @@ func TestPendingStream(t *testing.T) {
 		t.Fatalf("failed to create destination table(%q %q): %v", testDatasetID, testTableID, err)
 	}
 
-	if err := appendToPendingStream(ioutil.Discard, tc.ProjectID, testDatasetID, testTableID); err != nil {
-		t.Errorf("appendToPendingStream(%q %q): %v", testDatasetID, testTableID, err)
-	}
+	t.Run("PendingStream", func(t *testing.T) {
```

Comment:
comment: I'd like to move away from having nested  calls to `t.Run()` inside of a test case. That said, we don't yet have an official set of guidance around this, so I leave it up to you.

Reply:
The value here is that we share a common setup (reduces test time), but can more accurately pinpoint the error based on the generated test name, so the scope of failure is clearer if we do encounter failures.